index.html

```diff
@@ -11,6 +11,7 @@ <h1>Hello There Syntax Squad!</h1>
       <p>We would like you all to tryout GitHub Flow.</p>
       <p>Winnie and Jules 💋</p>
       <p>Welcome Ayo, Kelsey, Jose and Albert</p>
+      <p>THIS IS JOSE MAKING SOME CHANGES</p>
```

Comment:
This is great Jose! Just connfirming I'm happy if Winnie is

Reply:
great stuff!